Solutions/CiscoASA/Playbooks/CiscoASA-AddIPtoNetworkObjectGroup/readme.md

```diff
@@ -23,7 +23,7 @@ When a new Sentinel incident is created,this playbook gets triggered and perform
 
 
 ### Prerequisites
-1. **This playbook template is based on Azure Sentinel Incident Trigger which is currently in Private Preview (Automation Rules).** You can change the trigger to the Sentinel Alert trigger in cases you are not part of the Private Preview.
+1. **This playbook template is based on microsoft sentinel Incident Trigger which is currently in Private Preview (Automation Rules).** You can change the trigger to the Sentinel Alert trigger in cases you are not part of the Private Preview.
```

Comment:
Please caps initials letter of microsoft and sentinel work. eg: Microsoft Sentinel.